### PR TITLE
Code of Conduct update

### PR DESCRIPTION
**Pull request by krisfreedain**
### Description
The project is involved with more in-person events, and we should update to the Code of Conduct to easily reflect that. I spoke this over with legal, and they agreed this would be a welcome addition. 

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by peternied — APPROVED** [review]
Seems pretty reasonable to include events as well

**Comment by peternied** [comment]
@mch2 @saratvemulapalli Could we get a second review on this change?

**Review by dblock — APPROVED** [review]

**Pull request merged**
